django_celery_beat/schedulers.py

```diff
@@ -142,12 +142,19 @@ def _unpack_fields(cls, schedule,
                        args=None, kwargs=None, relative=None, options=None,
                        **entry):
         model_schedule, model_field = cls.to_model_schedule(schedule)
+        schedules = {
+            'interval': None,
+            'crontab': None,
+            'solar': None
+        }
+        schedules[model_field] = model_schedule
+
+        entry.update(**schedules)
         entry.update(
```

Comment:
So this will run two queries instead of one now?  It would be nicer to keep it as one.